kore/test/Test/Kore/Step/EquationalStep.hs

```diff
@@ -513,29 +515,38 @@ test_applyEquationalRule_ =
         EqualityRule RulePattern
             { left = Mock.a
             , antiLeft = Nothing
-            , right = mkBottom Mock.testSort
             , requires = makeTruePredicate_
-            , ensures = makeTruePredicate_
+            , rhs = RHS
```

Comment:
We should have a helper like `RHS.fromTermLike :: TermLike variable -> RHS variable` with `existentials = [], ensures = makeTruePredicate_`, because that pattern seems to occur often. That will make future changes easier.

Reply:
I've added a helper named `injectTermIntoRHS`